crates/cairoc/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "hieratika-cairoc"
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license-file.workspace = true
+
+authors.workspace = true
+description = "Generate the FLO object from Cairo files."
+keywords.workspace = true
+categories.workspace = true
+
+edition.workspace = true
+rust-version.workspace = true
+
+[dependencies]
+cairo-lang-compiler = { path = "../../cairo/crates/cairo-lang-compiler" }
+cairo-lang-defs = { path = "../../cairo/crates/cairo-lang-defs" }
+cairo-lang-filesystem = { path = "../../cairo/crates/cairo-lang-filesystem" }
+cairo-lang-lowering = { path = "../../cairo/crates/cairo-lang-lowering" }
+cairo-lang-semantic = { path = "../../cairo/crates/cairo-lang-semantic" }
+cairo-lang-utils = { path = "../../cairo/crates/cairo-lang-utils" }
```

Comment:
Please hoist whichever ones of these are likely to be needed by the code generator to workspace-level.